Source/Header/SwipeLabelView.swift

```diff
@@ -7,7 +7,13 @@ public final class SwipeLabelView: UIView, DayViewStateUpdating {
     case Backward
   }
 
-  public private(set) var calendar = Calendar.autoupdatingCurrent
+  public var calendar = Calendar.autoupdatingCurrent {
+    didSet {
+      // I doubt that someone will use this view without a state, but
+      // I consider it necessary to update text label for such a situation
+      updateLabelText()
```

Comment:
Yes, it's correct approach. `SwipeLabelView` should allow to be used without other modules, e.g. when the developer would like to create a custom view.